<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetFileRepair</class>
 <widget class="QWidget" name="WidgetFileRepair">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>451</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Res.qrc">:/system_scan/find_ico</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>451</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter your file name here. E.g. kernel32.dll</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_download">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>30</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>download</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>701</width>
     <height>441</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>download file:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delect">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>560</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>delete</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Res.qrc"/>
 </resources>
 <connections/>
</ui>
